src/libs/portfolio/helpers.ts

```diff
@@ -105,9 +206,10 @@ export const mapToken = (
   token: Pick<TokenResult, 'amount' | 'decimals' | 'name' | 'symbol'>,
   network: Network,
   address: string,
-  opts: Pick<GetOptions, 'specialErc20Hints'>
+  opts: Pick<GetOptions, 'specialErc20Hints' | 'blockTag'>,
+  hasSimulationAmount?: boolean
 ) => {
-  const { specialErc20Hints } = opts
+  const { specialErc20Hints, blockTag } = opts
```

Comment:
blockTag is not needed. latest tokens don't have simulation amounts as the simulation is done on the pending block